packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -51,7 +51,7 @@ const AdvertiserPage = () => {
         advertiser_page_store.onMount();
         advertiser_page_store.setIsDropdownMenuVisible(false);
 
-        return reaction(
+        reaction(
```

Comment:
`nit`: Should we assign this to a variable to dispose this reaction later on cleanup?